package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paypal/checkout-components",
-  "version": "5.0.289",
+  "version": "5.0.290",
```

Comment:
TODO: This must be removed before we merge. We have to increment for NPM alpha version testing